slider.css

```diff
@@ -0,0 +1,64 @@
+#carousel {
+    margin: auto;
+    padding-top: 50px;
+    max-width: 940px;
+	text-align: center;
+	font-size: 32px;
+    position: relative;
+    background-color: rgba(240,108,100,1);
+}
+.item {
+    margin: 0 auto;
+    width: 100%;
+	overflow: hidden;
+    height: 600px;
+	display: flex;
+	justify-content: center;
+    align-items: center;
+    background-color: rgba(240,108,100,1);
+}
+.item-1{
+    background: url(/singolo/images/slide-1.png) no-repeat;
+    background-size: contain;
+    background-position: center;
```

Comment:
Can it be applied in .item? As it's used both in .item-1 and item-2.
    background-size: contain;
    background-position: center;

Reply:
fixed